packages/ui/src/partials/index.ts

```diff
@@ -3,3 +3,4 @@ export * from './card'
 export * from './checkbox'
 export * from './input'
 export * from './label'
+export * from './react-form/form'
```

Comment:
I would like a dedicated folder for the forms. Not the partials.

Reply:
Sure, we can do it